code/modules/mob/living/living.dm

```diff
@@ -1396,6 +1396,25 @@
 		eminence.selected_mob = src
 		to_chat(eminence, "<span class='brass'>You select [src].</span>")
 
+/mob/living/proc/set_gender(ngender = NEUTER, silent = FALSE, update_icon = TRUE, forced = FALSE)
+	var/bender = gender != ngender
+	if(forced)
+		gender = ngender
+	if(bender)
```

Comment:
This doesn't seem to properly account for forced.
Also more appropriate variable names should be used, as oppossed to bender, I think

Reply:
Should forced not also update the model?

Reply:
Shifted stuff around, it now updates the model in the new place it does that stuff in.